src/main/java/org/jabref/gui/LibraryTab.java

```diff
@@ -558,21 +551,21 @@ public void setupMainPanel() {
     }
 
     /**
-     * Set up auto completion for this database
+     * Set up auto-completion for this database
      */
     private void setupAutoCompletion() {
         AutoCompletePreferences autoCompletePreferences = preferencesService.getAutoCompletePreferences();
         if (autoCompletePreferences.shouldAutoComplete()) {
             suggestionProviders = new SuggestionProviders(getDatabase(), Globals.journalAbbreviationRepository, autoCompletePreferences);
         } else {
-            // Create empty suggestion providers if auto completion is deactivated
+            // Create empty suggestion providers if auto-completion is deactivated
```

Comment:
Dashes are uncommon in English. Use "autocompletion".

(My dictionary hint: https://en.wikipedia.org/wiki/Autocomplete)

Reply:
ok, was a suggestion by the ide, but im completly unemotional about this